monkestation/code/modules/admin/verbs/commandreport.dm

```diff
@@ -0,0 +1,18 @@
+/datum/command_report_menu
+	var/append_update_name = TRUE
+	var/custom_played_sound
+
+/datum/command_report_menu/ui_static_data(mob/user)
+	. = ..()
+	.["append_update_name"] = append_update_name
+
+/datum/command_report_menu/ui_data(mob/user)
+	. = ..()
+	.["append_update_name"] = append_update_name
```

Comment:
```suggestion
/datum/command_report_menu/ui_data(mob/user)
	. = ..()
	.["append_update_name"] = append_update_name
```

you don't need the same thing in both `ui_static_data` and `ui_data`

`ui_static_data` is for data that isn't likely to be changed/won't change at all, or is expensive to update.
`ui_data` automatically updates.